op-devstack/sysgo/system.go

```diff
@@ -95,27 +95,32 @@ type DefaultInteropSystemIDs struct {
 
 	L2AProposer stack.L2ProposerID
 	L2BProposer stack.L2ProposerID
+
+	L2ChallengerA stack.L2ChallengerID
+	L2ChallengerB stack.L2ChallengerID
 }
 
 func NewDefaultInteropSystemIDs(l1ID, l2AID, l2BID eth.ChainID) DefaultInteropSystemIDs {
 	ids := DefaultInteropSystemIDs{
-		L1:          stack.L1NetworkID(l1ID),
-		L1EL:        stack.L1ELNodeID{Key: "l1", ChainID: l1ID},
-		L1CL:        stack.L1CLNodeID{Key: "l1", ChainID: l1ID},
-		Superchain:  "main", // TODO(#15244): hardcoded to match the deployer default ID
-		Cluster:     "main",
-		Supervisor:  "1-primary", // prefix with number for ordering of supervisors
-		Sequencer:   "dev",
-		L2A:         stack.L2NetworkID(l2AID),
-		L2ACL:       stack.L2CLNodeID{Key: "sequencer", ChainID: l2AID},
-		L2AEL:       stack.L2ELNodeID{Key: "sequencer", ChainID: l2AID},
-		L2B:         stack.L2NetworkID(l2BID),
-		L2BCL:       stack.L2CLNodeID{Key: "sequencer", ChainID: l2BID},
-		L2BEL:       stack.L2ELNodeID{Key: "sequencer", ChainID: l2BID},
-		L2ABatcher:  stack.L2BatcherID{Key: "main", ChainID: l2AID},
-		L2BBatcher:  stack.L2BatcherID{Key: "main", ChainID: l2BID},
-		L2AProposer: stack.L2ProposerID{Key: "main", ChainID: l2AID},
-		L2BProposer: stack.L2ProposerID{Key: "main", ChainID: l2BID},
+		L1:            stack.L1NetworkID(l1ID),
+		L1EL:          stack.L1ELNodeID{Key: "l1", ChainID: l1ID},
+		L1CL:          stack.L1CLNodeID{Key: "l1", ChainID: l1ID},
+		Superchain:    "main", // TODO(#15244): hardcoded to match the deployer default ID
+		Cluster:       stack.ClusterID("main"),
+		Supervisor:    "1-primary", // prefix with number for ordering of supervisors
+		Sequencer:     "dev",
+		L2A:           stack.L2NetworkID(l2AID),
+		L2ACL:         stack.L2CLNodeID{Key: "sequencer", ChainID: l2AID},
+		L2AEL:         stack.L2ELNodeID{Key: "sequencer", ChainID: l2AID},
+		L2B:           stack.L2NetworkID(l2BID),
+		L2BCL:         stack.L2CLNodeID{Key: "sequencer", ChainID: l2BID},
+		L2BEL:         stack.L2ELNodeID{Key: "sequencer", ChainID: l2BID},
+		L2ABatcher:    stack.L2BatcherID{Key: "main", ChainID: l2AID},
+		L2BBatcher:    stack.L2BatcherID{Key: "main", ChainID: l2BID},
+		L2AProposer:   stack.L2ProposerID{Key: "main", ChainID: l2AID},
+		L2BProposer:   stack.L2ProposerID{Key: "main", ChainID: l2BID},
+		L2ChallengerA: "chainA",
+		L2ChallengerB: "chainB",
```

Comment:
Just confirming, is hardcoding these OK? They stand out here.

Reply:
Yeah they're pretty arbitrary. Normally we'd make them `main` but since we need one for each chain currently we need something else. They just need to be unique.